Firestore/core/src/firebase/firestore/model/document_key.h

```diff
@@ -59,6 +59,10 @@ class DocumentKey {
   std::string ToString() const {
     return path().CanonicalString();
   }
+
+  uint64_t Hash() const {
```

Comment:
This return type doesn't make much sense. The two return values we use for hash codes today are:
  * `NSUInteger` for Objective-C hash methods
  * `size_t` for C++ specializations of `std::hash`.

Neither of these are fixed-size types. This should probably return `size_t`.

Reply:
done